src/test/java/uk/gov/justice/digital/delius/transformers/ReferralTransformerTest.java

```diff
@@ -5,33 +5,33 @@
 import java.util.UUID;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.*;
 import static uk.gov.justice.digital.delius.transformers.ReferralTransformer.prefixReferralStartNotesWithUrn;
 import static uk.gov.justice.digital.delius.transformers.ReferralTransformer.transformReferralIdToUrn;
 
 class ReferralTransformerTest {
-
     @Test
     public void generatesAUrnFromAReferralId() {
-
         final var referralId = UUID.randomUUID();
         assertThat(transformReferralIdToUrn(referralId))
-            .isEqualTo("urn:hmpps:interventions-referral:" + referralId.toString());
+            .isEqualTo("urn:hmpps:interventions-referral:" + referralId);
     }
 
     @Test
-    public void prefixesNotesWithReferralIdUrn() {
+    public void generatesNullURN_WhenReferralIdIsMissing() {
+        assertThat(transformReferralIdToUrn(null))
+            .isEqualTo(null);
+    }
 
+    @Test
+    public void prefixesNotesWithReferralIdUrn() {
         final var referralId = UUID.randomUUID();
         assertThat(prefixReferralStartNotesWithUrn("some notes", referralId))
-            .isEqualTo("urn:hmpps:interventions-referral:" + referralId.toString() + "\nsome notes");
+            .isEqualTo("urn:hmpps:interventions-referral:" + referralId + "\nsome notes");
     }
 
     @Test
-    public void doesNotPrefixesNotesWithUrn_WhenReferralIdNotSupport() {
-
-        final var referralId = UUID.randomUUID();
+    public void doesNotPrefixesNotesWithUrn_WhenReferralIdIsMissing() {
```

Comment:
(not a blocker)
As far as I know, we always have a referral UUID. @mxco86 Do you remember why we have this case?